resource_booking_simultaneous_human/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+When a resource of type human is scheduled for a booking, they remain
+simultaneously available for other bookings.
```

Comment:
Is this sentence a fact without this module, or the goal of this one?

Reply:
Added the word 'will'.